awx/main/utils/common.py

```diff
@@ -208,7 +208,6 @@ class StubLicense(object):
         'surveys': True,
         'system_tracking': True,
         'rebranding': True,
-        'enterprise_auth': True,
```

Comment:
For purposes of API compatibility, we _might_ want to keep this line around.

Reply:
Oops, this has been reverted.